docs/docsite/rst/dev_guide/developing_collections_creating.rst

```diff
@@ -22,16 +22,18 @@ A user can then install your collection on their systems.
 Creating a collection skeleton
 ==============================
 
-To start a new collection:
+To start a new collection, run the following command in your collections directory:
 
 .. code-block:: bash
 
-    collection_dir#> ansible-galaxy collection init my_namespace.my_collection
+    ansible_collections#> ansible-galaxy collection init my_namespace.my_collection
 
 .. note::
 
 	Both the namespace and collection names use the same strict set of requirements. See `Galaxy namespaces <https://galaxy.ansible.com/docs/contributing/namespaces.html#galaxy-namespaces>`_ on the Galaxy docsite for those requirements.
 
+It will create the structure ``[my_namespace]/[my_collection]/[collection skeleton]``.
+.. hint:: The git project should be initialized in the collection directory.
```

Comment:
Have you checked if it's rendered correctly, I'm pretty sure it needs empty lines around it like `.. note::` above:
```suggestion

.. hint::

   The git project should be initialized in the collection directory.

```

Reply:
Anyway, I have updated the hint with your suggestion (but no empty lines added)